crates/rnote-engine/src/pens/selector/penevents.rs

```diff
@@ -180,9 +180,16 @@ impl Selector {
                             };
                         } else {
                             // when clicking outside the selection bounds, reset
+                            tracing::debug!("cancelling the selection");
                             engine_view.store.set_selected_keys(selection, false);
                             self.state = SelectorState::Idle;
 
+                            // we create a new engine variable that indicates that the pen event is one from the same
+                            // event sequence that the one that cancelled the selection
+                            // this is done by setting this variable to true here and
+                            // removing it on a pen up event (and doing more things if this variable is true on a pen up)
```

Comment:
this comment should probably be modified and explain the cancelled_state variable in the engine struct definition, rather than having it here.

Reply:
I modified the comment here and moved the rest in the engine struct definition.

I also did a little renaming (the getters/setters name weren't matching the variable name in the engine struct definition)